tests/Resources/Files/AppsInfoDictionary.json

```diff
@@ -158,8 +158,8 @@
                          "matching": ""
                    },
                    "2": {
-                         "text": "https://console.redhat.com/application-services/streams/kafkas",
-                         "url": "https://console.redhat.com/application-services/streams/kafkas",
+                         "text": "https://cloud.redhat.com/application-services/streams/kafkas",
+                         "url": "https://cloud.redhat.com/beta/application-services/streams/kafkas",
```

Comment:
console should be the right one

Reply:
I think currently "cloud" is redirected automatically to "console"